### ISSUE TITLE
Keyboard interrupt not handled when waiting for user input

### ISSUE THREAD

**Issue by jose1711** [anufrievroman/calcure#67]
To reproduce:
* start calcure
* hit <kbd>a</kbd> to add event (but note that <kbd>d</kbd> or <kbd>r</kbd> works too)
* press <kbd>ctrl</kbd>+<kbd>c</kbd>

Receive `KeyboardInterrupt` message:
```
..
  File "/usr/lib/python3.11/site-packages/calcure/__main__.py", line 1066, in cli
    curses.wrapper(main)
  File "/usr/lib/python3.11/curses/__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/calcure/__main__.py", line 1025, in main
    control_monthly_screen(stdscr, screen, user_events, importer)
  File "/usr/lib/python3.11/site-packages/calcure/controls.py", line 39, in inner
    confirmed = ask_confirmation(stdscr, MSG_EXIT, cf.ASK_CONFIRMATIONS)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/calcure/dialogues.py", line 99, in ask_confirmation
    key = stdscr.getkey()
          ^^^^^^^^^^^^^^^
_curses.error: no input
```

**Comment by anufrievroman** [comment]
Indeed, I was aware of this possibility for a while. I think I should write an error handling decorator in `dialogues.py` module.

**Comment by jose1711** [comment]
Thanks, I believe user should be given an option to use <kbd>Ctrl</kbd>+<kbd>c</kbd> as a way of terminating the current action. Example:
- user hits <kbd>x<kbd>
- status line: `Delete event number: (^c to abort)`
- user realizes error, hits <kbd>ctrl</kbd>+<kbd>c</kbd>
- nothing happens, status line updates to `Aborted`

**Comment by anufrievroman** [comment]
Fixed it with commit 013b5202cac90ed98d7edf383d4385eb0cbff398
